src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1585,6 +1588,7 @@
     <Field Name="Expanded" Type="bool"/>
     <Field Name="InvokedAsExtensionMethod" Type="bool"/>
     <Field Name="ArgsToParamsOpt" Type="ImmutableArray&lt;int&gt;" Null="allow"/>
+    <Field Name="DefaultArgumentsOpt" Type="BitVector" Null="allow" />
```

Comment:
>DefaultArgumentsOpt [](start = 17, length = 19)

Consider naming `DefaultArguments` rather than `DefaultArgumentsOpt` since the value is always included. Same comment for other nodes.